VideoWeb/VideoWeb/Controllers/EndpointsController.cs

```diff
@@ -41,7 +41,7 @@ public class EndpointsController : ControllerBase
         [HttpGet("{conferenceId}/participants")]
         [SwaggerOperation(OperationId = "GetVideoEndpointsForConference")]
         [ProducesResponseType(typeof(List<VideoEndpointResponse>), (int)HttpStatusCode.OK)]
-        [Authorize(AppRoles.JudgeRole)]
+        [Authorize("Host")]
```

Comment:
Add this as a const in AppRoles

Reply:
these are not roles they are policy key's